samples/beam/requirements.txt

```diff
@@ -1,3 +1,3 @@
 apache-beam==2.31.0
 google-cloud-bigtable<2.0.0
-google-cloud-core==1.7.2
\ No newline at end of file
+google-cloud-core==2.0.0
```

Comment:
`google-cloud-bigtable < 2.0.0` is not compatible with `google-cloud-core 2.0.0`.